### ISSUE TITLE
Install fleet-x failed

### ISSUE THREAD

**Issue by HerbertArthur** [PaddlePaddle/PaddleFleetX#253]
I am using `pip install fleet-x` to install "fleet-x", but it failed and got this log:
```shell
(paddle) [root@2b4b7f89d99c ~]# pip install fleet-x
Looking in indexes: https://mirrors.aliyun.com/pypi/simple/
Collecting fleet-x
  Using cached https://mirrors.aliyun.com/pypi/packages/68/59/d435442e36543f39e753d88d81bfaf897559dcf655bc8ed91976c41e76d6/fleet-x-0.0.2.tar.gz (19 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/local/envs/paddle/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-of86qle5/fleet-x/setup.py'"'"'; __file__='"'"'/tmp/pip-install-of86qle5/fleet-x/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-nl8satpk
         cwd: /tmp/pip-install-of86qle5/fleet-x/
    Complete output (9 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-of86qle5/fleet-x/setup.py", line 25, in <module>
        from fleetx.version import fleetx_version
      File "/tmp/pip-install-of86qle5/fleet-x/fleetx/__init__.py", line 16, in <module>
        from . import applications
      File "/tmp/pip-install-of86qle5/fleet-x/fleetx/applications/__init__.py", line 15, in <module>
        from model import *
    ModuleNotFoundError: No module named 'model'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
Wishing someone to help me, thanks!

**Comment by HerbertArthur** [comment]
I've tried on two servers, it's both failed. The env of the one of them as follows:
```shell
(paddle) [root@2b4b7f89d99c ~]# pip list 
Package         Version
--------------- -------------------
astor           0.8.1
certifi         2020.6.20
chardet         3.0.4
click           7.1.2
cycler          0.10.0
decorator       4.4.2
funcsigs        1.0.2
gast            0.4.0
graphviz        0.14.1
idna            2.10
joblib          0.16.0
kiwisolver      1.2.0
matplotlib      3.3.1
nltk            3.5
numpy           1.19.1
objgraph        3.4.1
opencv-python   4.4.0.42
paddlepaddle    1.8.0
pathlib         1.0.1
Pillow          7.2.0
pip             20.2.2
prettytable     0.7.2
protobuf        3.13.0
pyparsing       2.4.7
python-dateutil 2.8.1
PyYAML          5.3.1
rarfile         4.0
regex           2020.7.14
requests        2.24.0
scipy           1.3.1
setuptools      49.6.0.post20200814
six             1.15.0
tqdm            4.48.2
urllib3         1.25.10
wheel           0.34.2
```

**Comment by qjing666** [comment]
Which version of python are you using?

**Comment by HerbertArthur** [comment]
> 
> 
> Which version of python are you using?

python 3.7.7

**Comment by qjing666** [comment]
I recommend you to use fleetx with python2.7 first, and I will fix the bug shortly.

**Comment by HerbertArthur** [comment]
> 
> 
> I recommend you to use fleetx with python2.7 first, and I will fix the bug shortly.

But i am using paddlepaddle==1.8 too, python 2.7 is incompatibility. Please hurry up to solve this problem.

**Comment by qjing666** [comment]
PaddlePaddle is also supported in python2.7

**Comment by qjing666** [comment]
bug fixed, please try fleet-x==0.0.3

**Comment by HerbertArthur** [comment]
> 
> 
> bug fixed, please try fleet-x==0.0.3

This bug was fixed, thanks.
